pkg/storage/apis/obot.obot.ai/group.go

```diff
@@ -0,0 +1,3 @@
+package obotobotplatformai
+
+var Group = "obot.obot.ai"
```

Comment:
How do we feel about this group?

Reply:
Don't love the stutter, but I'm not sure if `obot.ai` or `core.obot.ai` are any better.

Reply:
The other thing I considered was `obot.obot-platform.ai`. I switched it to this because we actually own `obot.ai`.

Reply:
kind of weird to have two obot....but I have no opinion